.github/workflows/build.yaml

```diff
@@ -49,4 +49,4 @@ jobs:
       - name: creds
         run: gcloud auth configure-docker --quiet
       - name: container
-        run: echo -n "${{secrets.COSIGN_PASSWORD}}" | KO_DOCKER_REPO=gcr.io/projectsigstore/cosign/ci/cosign make sign-container
+        run: echo -n "${{secrets.COSIGN_PASSWORD}}" | KO_PREFIX=gcr.io/projectsigstore/cosign/ci make sign-container
```

Comment:
I tried locally this and looks like this Env vars does not exist for `ko` -> `KO_PREFIX`

Reply:
Fixed!